requirements.txt

```diff
@@ -1,8 +1,8 @@
 torch==1.10.0
-allennlp==0.8.4
-python-Levenshtein==0.12.1
-transformers==4.11.3
-scikit-learn==0.20.0
-sentencepiece==0.1.95
+allennlp==0.9.0
```

Comment:
I know you mentioned this but just for the record, I would like to confirm that upgrading to 0.9.0 has no impact on gector's results?

Reply:
@damien2012eng did you see this message?